gov.pnnl.goss.gridappsd/src/gov/pnnl/goss/gridappsd/simulation/SimulationProcess.java

```diff
@@ -153,8 +158,19 @@ public void run() {
 						GridAppsDConstants.topic_platformLog);
 
 
-				//Send the timesteps by second for the amount of time specified in the simulation config
-                sendTimesteps(gossFncsResponseEvent, simulationConfig, simulationId);
+				//Send the start simulation command to the fncsgossbridge
+                startSimulation(gossFncsResponseEvent, simulationConfig, simulationId);
+                while(!isFinished.isFinished){
```

Comment:
really....isFinished.isFinished...can we get some better variable names.

Reply:
look at isInitialized.isInited